arrow/arrow.py

```diff
@@ -588,7 +588,28 @@ def interval(cls, frame, start, end, interval=1, tz=None, bounds="[)"):
     # representations
 
     def __repr__(self):
-        return f"<{self.__class__.__name__} [{self.__str__()}]>"
+
+        attrs = [
+            self.year,
+            self.month,
+            self.day,
+            self.hour,
+            self.minute,
+        ]
+
+        if any([self.second, self.microsecond]):
+            attrs.append(self.second)
+            attrs.append(self.microsecond)
+
+        attrs.append(f"tzinfo={self.tzinfo}")
+
+        if self.fold:
+            # only show if fold=1
+            attrs.append(f"fold={self.fold}")
+
+        attr_str = ", ".join(map(str, attrs))
+
+        return f"{__package__}.{self.__class__.__name__}({attr_str})"
```

Comment:
We should mimic the `datetime.datetime.__repr__` by doing `f"{self.__class__.__module__}, {self.__class__.__qualname__}"`:

https://github.com/python/cpython/blob/7c9487ded487f304c2906698c52f0815b92cbeb6/Lib/datetime.py#L1924-L1941

Reply:
That results in too many arrows! :sweat_smile: 
```shell
>>> arw=arrow.utcnow()
>>> arw
arrow.arrow.Arrow(2020, 11, 19, 19, 18, 7, 53415, tzinfo=tzutc())
```

Reply:
Hah yeah, does `self.__class__.__qualname__` return `arrow.Arrow`?